src/QueryAutomater.py

```diff
@@ -0,0 +1,19 @@
+import requests
+
+def query_automater():
+    request_data = requests.get('http://www.mocky.io/v2/5e4ddd0a2f00002b0016a28a')
```

Comment:
We can remove this mock url that we have used for testing. Instead, we can read a json in a caller method and pass the keywords as a param so that we can even support different set of keywords for different queries